client/lib/cart-values/cart-items.js

```diff
@@ -615,7 +621,7 @@ function getDomainRegistrationsWithoutPrivacy( cart ) {
  */
 function changePrivacyForDomains( cart, domainItems, changeFunction ) {
 	return flow.apply( null, domainItems.map( function( item ) {
-		return changeFunction( domainPrivacyProtection( { domain: item.meta } ) );
+		return changeFunction( domainPrivacyProtection( { domain: item.meta, volume: item.volume } ) );
```

Comment:
Might be a decent time to add some destructuring and spread to cut down on the typing

```js
return flow( ...domainItems.map(
	( { meta: domain, volume } ) => changeFunction( domainPrivacyProtection( { domain, volume } ) )
) );
```

Actually, we might have an issue with the spread operator, in which case if that's true you should stick with `apply`